testnet/stacks-node/src/config.rs

```diff
@@ -1958,7 +1958,7 @@ impl Default for NodeConfig {
             max_microblocks: u16::MAX as u64,
             wait_time_for_microblocks: 30_000,
             wait_time_for_blocks: 30_000,
-            next_initiative_delay: 10_000,
+            next_initiative_delay: 1_000,
```

Comment:
Why change this delay? This basically controls how often a miner checks if they should RBF an outstanding block commit. That check isn't free: it's not too burdensome either, but every second could be a lot of wasted energy.

Reply:
That's left over from when I was futzing with the `initiative()` loop.  I'll revert